docs/modules/deployment/pages/upgrade/git-diff.adoc

```diff
@@ -0,0 +1,116 @@
+[[git-diff]]
+= Manage Configuration Changes with Git
+
+This tutorial was originally written on the OpenNMS Discourse forum (see https://opennms.discourse.group/t/managing-configuration-changes-with-git/2327[Managing Configuration Changes with Git]).
+
+If you want to manage your configuration files with git but haven't used it for version control before, GitHub's https://docs.github.com/en/get-started/using-git[Using Git] tutorial covers the basics.
+You can also read https://opennms.discourse.group/t/developing-with-git/2228[Developing with Git] on the OpenNMS Discourse forum.
+
+If you have an existing OpenNMS {page-component-title} installation, check for the `etc-pristine` directory.
+If your {page-component-title} environment does not have this directory, you will need to upgrade to a version that does.
+
+IMPORTANT: Remember to stop {page-component-title} before changing files in the `etc-pristine` directory.
+
+== Start with pristine `etc` directory
+
+[{tabs}]
+====
+Debian OSes::
++
+--
+IMPORTANT: `$OPENNMS_HOME/etc` is linked to `/etc/opennms`.
+Do not move that folder.
+Moving the folder will break the link and will cause future upgrades to fail.
+
+You can fix this by running the following code:
```

Comment:
Not clear on what they are fixing. The fact that it broke, or how to make a copy of the folder rather than moving it?

Reply:
I think it's fixing the link between the two folders? 

@mmahacek, could you clarify here? :)

Reply:
Does this look like an okay disclaimer to cover the workaround?

```suggestion
You can prevent this by running the following code (note that this is a workaround and does not remove the link between the folders):
```

Reply:
```suggestion
You can fix this by running the following code:
```
Since we've moved the word "fix" I don't think we need to state that it's a workaround.